packages/apollo-server-core/src/__tests__/ApolloServerBase.test.ts

```diff
@@ -30,6 +30,50 @@ describe('ApolloServerBase construction', () => {
     ).not.toThrow();
   });
 
+  it('succeeds when passed a graphVariant in construction', () => {
+    let serverBase;
+    expect(
+      () =>
+        new ApolloServerBase({
+          schema: buildServiceDefinition([{ typeDefs, resolvers }]).schema,
```

Comment:
Suggestion I should have suggeseted:

```suggestion
          typeDefs,
          resolvers,
```